Thrive.csproj

```diff
@@ -1,4 +1,3 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think that the changes to this file are not very good. Did some tool make these?
The removal of the xml version and encoding is worrying. Also both Godot and MonoDevelop use the convention of needing a space before a self-closing tag end ` />` and not `/>`

Reply:
Those were made by Nuget, except the include of System.Numerics.Vectors which i modified because it doesn't work by default.

Reply:
Could you undo those? Nuget packages can be manually added to the file without nuget mangling things.

Reply:
Whereas adding new files is done often, and that follows the style already in the file when added with Godot or with MonoDevelop.